src/main/java/views/TrainerFormViewController.java

```diff
@@ -87,6 +76,8 @@ public void subscribeDataChangeListener(DataChangeListener listener) {
   @FXML
   public void onButtonSaveAction(ActionEvent event) {
 
+    boolean saveOrUpdate;
```

Comment:
Se essa variável define se a tela vai salvar ou atualizar, a nomeclatura dela fica mais intuitiva se for `isUpdate`, se não for pra atualizar, é pra salvar

Reply:
Aí pra facilitar as teclas `shift + f6` no IntelliJ renomeiam a variável